tests/test_attribute_definition.py

```diff
@@ -17,6 +17,33 @@
 from xdsl.printer import Printer
 from xdsl.utils.exceptions import PyRDLAttrDefinitionError, VerifyException
 
+################################################################################
+# non-IRDL attributes
+################################################################################
+
+
+class NonIRDLAttr(ParametrizedAttribute):
```

Comment:
Feels awkward tbh. I guess this is pretty much defined by its verifier, right?

Reply:
Does that mean that, in a strict sense, you define everything as IRDL and non-IRDL? Looks a bit weird, but I do not have a strong opinion, I would just like to understand better why is this used.